pkg/agent/openflow/pipeline.go

```diff
@@ -128,10 +129,11 @@ var (
 	IPv6Table                 = newTable("IPv6", stageValidation, pipelineIP)
 	PipelineIPClassifierTable = newTable("PipelineIPClassifier", stageValidation, pipelineIP)
 
-	// Tables in stageConntrackState:
-	UnSNATTable         = newTable("UnSNAT", stageConntrackState, pipelineIP)
-	ConntrackTable      = newTable("ConntrackZone", stageConntrackState, pipelineIP)
-	ConntrackStateTable = newTable("ConntrackState", stageConntrackState, pipelineIP)
+	// Tables in stageConntrack:
+	UnSNATTable          = newTable("UnSNAT", stageConntrack, pipelineIP)
+	ConntrackTable       = newTable("ConntrackZone", stageConntrack, pipelineIP)
+	ConntrackCommitTable = newTable("ConntrackCommit", stageConntrack, pipelineIP)
```

Comment:
could the commit flow be added to ConntrackStateTable directly?

Reply:
It seems if we commit in ConntrackCommitTable or ConntrackStateTable, Service traffic will have one extra ct action, which may affect pod-to-Service TCP_CRR, could you test if it's the case? If yes, I think we should add one commit action in endpointDNAT table for unmatched new connections instead.

Reply:
It looks all kind tests are failed. Having check one failure on ANPReject, it is possibly because the connection is committed to conntrack before ANP check, and the reject packet sent from Agent is marked as ct_state=+rpl/+est when going through conntrack. Originally the reject packet is expected to be dropped by ct_state=+inv, and the forked packet by conntrack is output to the client. 

Another failed case is Traceflow, doubt a similar reason that the ct_state is unexpected.